fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -532,7 +532,8 @@ private void handleExecute() {
             // new_params_bind_flag
             if (packetBuf.hasRemaining() && (int) packetBuf.get() != 0) {
                 // parse params types
-                IntStream.range(0, numParams).forEach(i -> mysqlTypeCodes[i] = (int) packetBuf.getChar());
+                IntStream.range(0, numParams)
+                        .forEach(i -> prepareCtx.getStmt().addMysqlTypeCodes((int) packetBuf.getChar()));
```

Comment:
Why not use `Integer[] mysqlTypeCodes = new Integer[numParams];`?  `mysqlTypeCodes ` is no used and can be removed?

Add some comments plz.

Reply:
Integer[] mysqlTypeCodes = new Integer[numParams] Should be removed

Reply:
Why add elements into the same `mysqlTypeCodes` in `PrepareStmt`?  Is there any problem lead the list size larger than placeholder number?

Reply:
When executing the first execute, the packetBuf[18] is {[23, 1, 0, 0, 0, 1, 1, 0, 0, 0, 0, 1, 3, 0, 1, 0, 0, 0]}.

When executing the second execute, the packetBuf[16] is {[23, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 2, 0, 0, 0]}.

During the execution of the second execute, based on the content of the packetBuf package, the condition packetBuf.get() != 0 cannot be used to set mysqlTypeCodes. It is necessary to reuse the existing mysqlTypeCodes placeholders in the session prepareContextStmt

Reply:
> Why add elements into the same `mysqlTypeCodes` in `PrepareStmt`? Is there any problem lead the list size larger than placeholder number?

Yes, I'll take care of it.